subcommands/waves/init.go

```diff
@@ -135,6 +168,17 @@ func readExpiration(cmd *cobra.Command) (expires time.Time) {
 	return expires.Round(time.Second)
 }
 
+func readOfflineKeys(cmd *cobra.Command) keys.OfflineCreds {
+	offlineKeysFile, _ := cmd.Flags().GetString("keys")
+	if offlineKeysFile == "" {
```

Comment:
things like this are why i hate cobra. i'm sure there's a better way but the API leads you in bad directions. no worries, just ranting.

Reply:
your are the biggest foe of my laziness: 31255be